packages/tpetra/core/src/Tpetra_BlockCrsMatrix_decl.hpp

```diff
@@ -1234,13 +1234,13 @@ class BlockCrsMatrix :
   getGlobalRowCopy (GO GlobalRow,
                     nonconst_global_inds_host_view_type &Indices,
                     nonconst_values_host_view_type &Values,
-                    size_t& NumEntries) const;
+                    size_t& NumEntries) const override;
 #ifdef TPETRA_ENABLE_DEPRECATED_CODE
```

Comment:
I am assuming that these `#ifdef TPETRA_ENABLE_DEPRECATED_CODE` directives will be removed later on, right?

Reply:
Not until they go away from tpetra, we'll remove all of them at the same time